app/config.py

```diff
@@ -38,6 +40,8 @@ def int_or_none(key: str) -> Optional[int]:
             int(c) for c in env.pop("AUTOPUBLISH_USERS").split(" ")
         ]
 
+        self.join_messages = open("assets/join_messages").read().strip().split("\n")
```

Comment:
Change `.split("\n")` to `.splitlines()`, [here's why.](https://stackoverflow.com/a/22233816)